test/common.py

```diff
@@ -1934,7 +1934,7 @@ def do_run_in_out_file_test(self, srcfile, **kwargs):
     return output
 
   ## Does a complete test - builds, runs, checks output, etc.
-  def _build_and_run(self, filename, expected_output, args=None,
+  def _build_and_run(self, filename, /, expected_output, args=None,
```

Comment:
why allow `expected_output` positionally in one of the last two functions but not both?